CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+#Change Log
+
+
+#0.0.2 (October X, 2018)
+
+
+* Prolog completion on first line, by typing: ```<?xml...``` . See [#85](https://github.com/angelozerr/lsp4xml/issues/85).
+
+* Definition on hover for schema defined attributes. See [#146](https://github.com/angelozerr/lsp4xml/issues/146).
+
+* Ability to locally cache schemas (on by default). See [#159](https://github.com/angelozerr/lsp4xml/issues/159).
+
+* Autocompletion and Hovers for XSD's. See [#111](https://github.com/angelozerr/lsp4xml/issues/111)
```

Comment:
right now I don't get hover for xsds. See https://github.com/angelozerr/lsp4xml/issues/111#issuecomment-434803478